ci/smoke-tests-login.py

```diff
@@ -9,7 +9,7 @@
 if __name__ == "__main__":
     session = requests.Session()
 
-    logs = session.get('https://logs.{}'.format(os.environ['CF_SYSTEM_DOMAIN']))
+    logs = session.get('https://logs-deprecated.{}'.format(os.environ['CF_SYSTEM_DOMAIN']))
```

Comment:
We'll have to pause prod to make sure these changes don't go there until we intend them to?

Reply:
Yes, its either that or we bypass in the piple